lib/addons/helm-addon/helm-version-checker.ts

```diff
@@ -67,14 +67,18 @@ export interface CheckVersionResult {
  */
 export function listChartVersions(chart: HelmChartVersion): string[] {
     // TODO make function async and use async HTTP client to get results
-    if(chart.repository.startsWith("oci:")){
-        return []; // skip for now if OCI repo
+    if (chart.repository) {
```

Comment:
I am unclear on what changed here? (vscode setting should be set to format modified lines only).